src/models/signer.cr

```diff
@@ -3,17 +3,22 @@ require "openssl"
 require "../constants"
 require "../error"
 
-class PassKit::Signer
+class Passkit::Signer
```

Comment:
Why the lowercasing?

Reply:
My bad, I updated this but forgot to push up the change